CONTRIBUTING.md

```diff
@@ -15,6 +15,7 @@ All contributors are expected to follow the [Rust Code of Conduct].
 - [Contributing to Clippy](#contributing-to-clippy)
   - [Getting started](#getting-started)
     - [Finding something to fix/improve](#finding-something-to-fiximprove)
+    - [How to build Clippy](#how-to-build-clippy)
```

Comment:
And finally: Please remove this line and squash your commits. After that this is good to go 👍

Reply:
Ah, I'm used to rust-lang/rust where bors auto-squashes :)

Reply:
Squashed!